### ISSUE TITLE
Clarifying the YOLO degrees Parameter: Range and Learning Implications

### ISSUE THREAD

**Issue by fcu52005505** [ultralytics/ultralytics#19978]
In the YOLO documentation, there is a degrees parameter with a range of -180 to 180. However, I want to configure it so that setting degrees=10 covers all angles between -10 and 10 degrees (for example, -1, 3, 7 degrees would be included), rather than only considering an angle of 10 degrees.

Additionally, I would like to confirm: when setting the angle to 10 degrees, will the model also learn from the original 0-degree image, or does setting it to 10 degrees mean it will only learn from images at 10 degrees?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @fcu52005505, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com/) for detailed explanations where many parameters, including `degrees`, are covered. You can find helpful examples for [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us investigate the issue further.

For your ❓ question about the `degrees` parameter, please include more details if relevant, such as:
- Your configuration file or training script
- Dataset examples
- Training logs

This will help us provide a more precise response!

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or join our [Subreddit](https://reddit.com/r/Ultralytics) to share insights and learn from the community.

## Upgrade

Ensure you’re using the latest `ultralytics` package with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). Run the following command to upgrade:

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in the following verified environments (with dependencies like [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response, but rest assured, an Ultralytics engineer will review your question and assist you further soon! 😊

**Comment by fcu52005505** [comment]
> > I want to configure it so that setting degrees=10 covers all angles between -10 and 10 degrees (for example, -1, 3, 7 degrees would be included), rather than only considering an angle of 10 degrees.
> 
> `degrees=10` does that already
> 
> > Additionally, I would like to confirm: when setting the angle to 10 degrees, will the model also learn from the original 0-degree image, or does setting it to 10 degrees mean it will only learn from images at 10 degrees?
> 
> `degrees=10` isn't just -10 or 10. It's the whole range.

As you pointed out, if setting degrees=10 already accounts for both positive and negative 10 degrees, why does the documentation define the range as -180 to 180 degrees instead of just 0 to 180?

**Comment by Y-T-G** [comment]
degrees=-10 and degrees=10 would do the same thing
